src/components.tsx

```diff
@@ -208,6 +208,11 @@ export interface VirtuosoProps<D> extends Omit<ListProps, 'groupCounts' | 'group
    */
   useWindowScroll?: boolean
 
+  /**
+   * Uses an external scroll element rather than wrapping the list in its own.
```

Comment:
```suggestion
   * Pass a reference to a scrollable parent element, so that the list won't wrap in its own.
```
Assuming you are a native speaker, you can certainly improve on that. I would like to make it obvious that 
- a `ref` to an HTMLElement should be passed;
- the ref should point to a parent element (kind of common sense, but still).